src/ScheduleService.Models.Dto/Requests/Category/Filters/GetCategoriesFilter.cs

```diff
@@ -0,0 +1,7 @@
+﻿using LT.DigitalOffice.Kernel.Requests;
+
+namespace LT.DigitalOffice.ScheduleService.Models.Dto.Requests.Category.Filters;
+
+public record GetCategoriesFilter : BaseFindFilter
```

Comment:
add bool IsAscendingSort to manage sort (asending/descending) and string IncludeSubstring to filter by category name

Reply:
+